test/it/group-get.js

```diff
@@ -31,13 +31,14 @@ describe('Group API tests', () => {
     const group = await client.getGroup(createdGroup.id);
     utils.validateGroup(group, createdGroup);
 
-    // 3. Get group stats
-    const stats = await createdGroup.getStats();
-    expect(stats.usersCount).to.equal(0);
-    expect(stats.appsCount).to.equal(0);
-    expect(stats.groupPushMappingsCount).to.equal(0);
-
-    // 4. Delete the group
+    // 3. Delete the group
     await client.deleteGroup(createdGroup.id);
+
+    // 4. Verify group was deleted
+    try {
+      await client.getGroup(createdGroup.id);
```

Comment:
In this case, since the group was deleted, an exception is thrown. If we don't catch it, the test would exit with failure. In other cases, we don't expect an exception. And if it is thrown, it might be a genuine bug.